### PR TITLE
KPI Widget: Min/Max Initial Checks

### PR DESCRIPTION
**Pull request by raul-gracia**
## Description
Refactor the initial checks widget to return a hash with the average, min, and max
values.
Also update the widget html to include this changes

## Trello Card Link

https://trello.com/c/3YTez2od

### CODE REVIEW

**Comment by raul-gracia** [comment]
[KPI Widget: Min/Max Initial Checks](https://trello.com/c/3YTez2od/101-kpi-widget-min-max-initial-checks)

File: app/queries/time_to_initial_checks_query.rb [hunk 1]
```diff
@@ -6,8 +6,12 @@ def initialize(relation = ApplicationProgress.all)
   def call
     applications_list = @relation.where.not(created_at: nil).where.not(initial_checks_completed_at: nil)
 
-    average_time = applications_list.average("initial_checks_completed_at::date - created_at::date")
+    durations = applications_list.map { |app| (app.initial_checks_completed_at.to_date - app.created_at.to_date).to_i }
```
Comment (padv2010) [inline app/queries/time_to_initial_checks_query.rb:9]: @raul-gracia would this work when dates are `nil`?

**Review by padv2010 — APPROVED** [review]
Can you please check that this would work with `nil` dates?

File: app/queries/time_to_initial_checks_query.rb (hunk 1 above)
Comment (raul-gracia) [reply app/queries/time_to_initial_checks_query.rb:9]: it should be fine, there are few checks after for nil values. But I will test it locally with no data.

**Pull request merged**